hw_icap_ip/README.md

```diff
@@ -0,0 +1,95 @@
+# Xilinx HWICAP IP

```

Comment:
Şimdi aslında ICAP birçok komut da atmanı sağlıyor ya ICAPE2 den bahsediyorum örneğin. Bu yazıda bundan  da bahsetsen iyi olabilir bence. Yani biz burada bir reset örneği yapacağız ama aslında şu dokümanı okursanız UGxxx başka komutlar da var, onları da HXICAP IP ile gönderebilirsiniz vs. diye

Reply:
Bu kısımla ilgili ekleme yaptım.